server-phoenix/lib/helios/announcement.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Helios.Announcement do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias Helios.Announcement
+  import Ecto.Query
+
+  @primary_key {:id, :id, autogenerate: true}
+  schema "announcements" do
+    field :publish_on, :utc_datetime
```

Comment:
The absinthe announcement type has this field as a `:datetime`, which is a built in scalar. I notice that they defined a `UTCDateTimeType` scalar on the rails end to use for the `publish_on` field. Then in schema.rb `publish_on` is a `t.datetime`. Do you know why the types are different in schema.rb and the graphql type definition (announcement_type.rb)?